### PR TITLE
[plugin.video.crackle] 2020.10.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Description

        - Update code for Python3
        - Improve Sorting of list items (thanks ssreekanth)
        - Add DRM playback support (Allows for HD streams)
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.crackle/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```
Comment (enen92) [inline plugin.video.crackle/.gitattributes]: please drop from submission. If you're automating submission try to add to .gitatttributes

File: plugin.video.crackle/.gitignore [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Windows image file caches
```
Comment (enen92) [inline plugin.video.crackle/.gitignore]: same

File: plugin.video.crackle/addon.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <summary lang="en_GB">Crackle delivers popular, award-winning TV, movies and originals. With no limit to how much you can watch across all your devices, you can binge all you want, wherever you want.</summary>
+    <description lang="en_GB">Crackle is a video streaming distributor of original web shows, Hollywood movies, and TV shows. Founded in the early 2000s as Grouper, and rebranded in 2007, Crackle is owned by Sony Pictures Entertainment.</description>
+      <news>
+        - Update code for Python3
+        - Improve Sorting of list items (thanks ssreekanth)
+        - Add DRM playback support
+      </news>
+    <language>en</language>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline plugin.video.crackle/addon.xml]: spdx identifier here if possible: GPL-2.0-only or GPL-2.0-or-later

**Review by enen92 — CHANGES_REQUESTED** [review]
Minors, after those fixes can be merged. Thanks for the update!

**Comment by eracknaphobia** [comment]
@enen92 Ok, I think I got everything. Thanks for the review!

**Review by enen92 — APPROVED** [review]

**Pull request merged**